tests/pkg_cryptoauthlib_compare_sha256/main.c

```diff
@@ -0,0 +1,85 @@
+/*
```

Comment:
Generally, this test seems helpful although I don't think we would merge it in this way. I don't know what would the best way to restructure it, but I'll think of it.